app/lib/features/home/widgets/my_spaces_section.dart

```diff
@@ -87,32 +82,28 @@ class _RenderSpacesSection extends ConsumerWidget {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.stretch,
       children: [
-        title,
+        Row(
+          children: [
+            title,
+            const Spacer(),
+            ActerInlineTextButton(
+              onPressed: () => context.pushNamed(Routes.spaces.name),
+              child: Text(L10n.of(context).seeAll),
+            ),
+          ],
+        ),
+        const SizedBox(height: 8),
```

Comment:
I wonder what all these rows look like when we have more narrow screens or large fonts... has this been thought about?